### PR TITLE
Makes cult structures breakable and readds cult recipes to runed metal

### PR DESCRIPTION
**Pull request by CHAD-BRO**
:cl: CHAD-BRO
add: Structure code from clockwork cult to all blood cult structures.
add: Cult structure recipes back to runed metal.
/:cl:

I'm not too experienced with BYOND or coding in general, but I just couldn't sit idle while nobody did anything about cult structures being invincible other than just removing them wholesale. So I used what I knew worked in game and copied the code from clockwork_structures to cult_structures and with a few changes to wording it ended up working.

There's currently no way to examine them to see how much health they have left or any way to repair them. 

It's not perfect but I figure this is a pretty good start to a problem nobody seems to want to address.

### CODE REVIEW

File: code/game/gamemodes/cult/cult_structures.dm [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 /obj/structure/cult
-	density = 1
-	anchored = 1
 	icon = 'icons/obj/cult.dmi'
-	var/cooldowntime = 0
+	anchored = 1
+	density = 1
+	opacity = 0
+	layer = OBJ_LAYER
+	var/max_health = 100 //All clockwork structures have health that can be removed via attacks
```
Comment (Jalleo) [inline code/game/gamemodes/cult/cult_structures.dm]: You could of removed the comment here.

**Comment by Kromgar** [comment]
Should add [Fix]

File: code/game/gamemodes/cult/cult_structures.dm [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 	var/health = 100
-	var/maxhealth = 100
+	var/takes_damage = TRUE //If the structure can be damaged
+	var/break_message = "<span class='warning'>The frog isn't a meme after all!</span>" //The message shown when a structure breaks
+	var/break_sound = 'sound/magic/clockwork/anima_fragment_death.ogg' //The sound played when a structure breaks
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/cult_structures.dm]: Might want a different sound but w/e

Comment (CHAD-BRO) [inline code/game/gamemodes/cult/cult_structures.dm]: I'm not really sure what would be the best replacement.

**Comment by AdamElTablawy** [comment]
h e r o

**Comment by FantasticFwoosh** [comment]
Real human bean

**Comment by AnturK** [comment]
Another separate structure health system. Ugh

**Comment by Cheridan** [comment]
Yeaah, this really shouldn't be copy-pasted. The bloodcult and clockcult structures should share a base type.

**Comment by optimumtact** [comment]
Yep, if you are unable to do that refactor please close this pr.

**Comment by CHAD-BRO** [comment]
Oh? I was actually supposed to do something more? Well I don't know shit about coding, but now that people know this is possible there's no excuse for removing this part of cult.

**Pull request closed without merging**